src/components/CodeSample.vue

```diff
@@ -3,7 +3,12 @@
     <div class="syn-code-block-header syn-flex justify-space-between align-center">
       <span class="syn-overline syn-text-light-primary syn-mb-0 syn-ml-16 syn-mr-8">{{title}}</span>
       <div class="syn-ma-8 syn-flex align-center">
-        <select class="syn-code-language-selector syn-mr-8" v-if="languages.length > 1" v-model="selectedLanguage">
+        <select
```

Comment:
Since we're updating our vue components with this PR, I would really appreciate it if the jest tests were working again-- makes me nervous committing changes to components without being able to be sure all the functionality is okay

Reply:
Done.